topics/lua-api.md

```diff
@@ -411,21 +405,32 @@ The named argument version except the following named arguments:
 * _flags_ - Lua array of strings representing function flags
 * _description_ - Lua string representing the function description
 
-_function\_name_ and _callback_ are mandatory, the other named arguments are option. Error raise on the following cases:
-
-* Missing mandatory arguments
-* Unkown named argument
-* Wrong argument type
-* Unknows flag
-* Another function with the same name already registered on the current library
+_function\_name_ and _callback_ are mandatory, the other named arguments are option.
 
 Usage Example:
 
 ```
 redis> FUNCTION LOAD Lua mylib "redis.register_function{function_name='noop', callback=function() end, function, flags={'no-writes'}, description='some desc')"
 ```
 
-For more information about function flags refer to [Function Flags](functions-intro#function-flags)
+#### <a name="script_flags"></a> Script Flags
+
+The following is a list of supported script flags:
```

Comment:
i think we should mention the default.
i.e. on functions, all these flags are off by default.
on eval... when shebang is used, it's the same, but if it's not used, it's a different story (maybe refer to another section)
@yoav-steinberg FYI